x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -226,9 +226,9 @@ primaryExpression
     ;
 
 builtinDateTimeFunction
-    : name=CURRENT_TIMESTAMP ('(' precision=INTEGER_VALUE? ')')?
-    | name=CURRENT_DATE ('(' ')')?
-    | name=CURRENT_TIME ('(' precision=INTEGER_VALUE? ')')?
+    : name=CURRENT_TIMESTAMP
```

Comment:
Please add them in alphabetical order.

Reply:
Those are in the order of priority/importance. There is an open issue to check and fix it everywhere: https://github.com/elastic/elasticsearch/issues/40790

Reply:
Since this is the grammar though, do you prefer the alphabetical ordering?